scripts/gn/README.md

```diff
@@ -12,8 +12,14 @@ cd /path/to/ginan/scripts
 # Create a new virtualenv
 python -m venv env
 
+# Activate the virtualenv
+. env/bin/activate
```

Comment:
Valid, though `source` is _reasonably_ universal and probably clearer.

Reply:
Sounds good